tilecloud/store/url.py

```diff
@@ -35,10 +35,13 @@ def get_one(self, tile):
             tile.content_type = response.headers.get("Content-Type")
             if response.status_code < 300:
                 tile.data = response.content
-                if tile.content_type.startswith("image/"):
-                    tile.data = response.content
-                else:
-                    tile.error = response.text
+                if tile.content_type:
+                    if tile.content_type.startswith("image/"):
+                        tile.data = response.content
+                    else:
+                        tile.error = response.text
+                elif response.status_code != 200:
+                    tile.error = "Unable to read content type {0}".format(title.content_type)
```

Comment:
I don't understand why you do this change? Shouldn't just be:
```suggestion
                if tile.content_type and tile.content_type.startswith("image/"):
                    tile.data = response.content
                else:
                    tile.error = response.text
```

Reply:
Sometimes the response has content but not content type header set so it continues failing, i tested it using openstreetmap

Reply:
Have you an example of URL that didn't returns a content-type?

Reply:
I just merge the first part, and for this I do an alternative as #673, does it look good?

Reply:
Wouldn't i it be better coding three commands, Ill make it up